src/views/workflow-queries/workflow-queries-loader/workflow-queries-loader.tsx

```diff
@@ -26,5 +29,49 @@ export default function WorkflowQueriesLoader(props: Props) {
       ).then((res) => res.json()),
   });
 
-  return <PageSection>{JSON.stringify(queryTypes)}</PageSection>;
+  const [selectedQueryIndex, setSelectedQueryIndex] = useState<number>(-1);
+  const [inputs, setInputs] = useState<Record<string, string | undefined>>({});
+
+  const queries = useQueries({
```

Comment:
useQueries accepts generic types which sets the responses type,  queryKey, Error types etc

Reply:
Yes, I'll add these generic types when adding the actual query logic. I can add a TODO here.